src/mocks/mockMediaQuery.ts

```diff
@@ -0,0 +1,15 @@
+const MatchMedia = Object.defineProperty(window, 'matchMedia', {
```

Comment:
I dont think you meant to check this in, or did you?

Reply:
Removed, thanks for spotting that